xbmc/network/GUIDialogNetworkSetup.cpp

```diff
@@ -219,9 +219,6 @@ void CGUIDialogNetworkSetup::InitializeSettings()
          { true,  true,  true,  true, false,  80,   "rss", 20304}};
 
  m_protocols.insert(m_protocols.end(), defaults.begin(), defaults.end());
-#ifdef HAS_FILESYSTEM_NFS
-  m_protocols.emplace_back(Protocol{true, false, false, false, true, 0, "nfs", 20259});
```

Comment:
is this correct?

Reply:
Yes but i have to finalize my work first. add-on shoukd carry the info and then we would append addon info here

Reply:
see https://github.com/xbmc/xbmc/pull/13038